source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/ChoiceSetInputRenderer.java

```diff
@@ -76,6 +76,7 @@ public View renderCheckBoxSet(
         List<CheckBox> checkBoxList = new Vector<CheckBox>();
         ChoiceInputVector choiceInputVector = choiceSetInput.GetChoices();
         long size = choiceInputVector.size();
```

Comment:
Do we have a sample card that you can check in to teh samples folder for the bug bash testing. Also can we add a unit test for this?